src/coreclr/src/jit/compiler.h

```diff
@@ -8270,7 +8270,7 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
         return false;
     }
 
-    bool compSupports(InstructionSet isa) const
+    bool compSupports(CORINFO_InstructionSet isa) const
```

Comment:
Seems like we should remove this method, retype `compSupportsISA` as `CORINFO_InstructionSetFlags`, and have consumers call the equivalent methods defined on that type. But this can be done as a follow-up.

Reply:
I believe the idea was to take the new names we discussed in the other WIP PR that I have, but I didn't want to merge all of that with this.